src/app/courses/courses-list/courses-list.component.html

```diff
@@ -1 +1,8 @@
-<p>courses-list works!</p>
+<ul class="coursesList">
+    <li *ngFor="let item of courses; trackBy: trackByFn">
+        <app-course-item [id]="item.Id" [title]="item.Title" [description]="item.Description" [duration]="item.Duration"
+            [creationDate]="item.CreationDate" (deletedItemId)="onDeleteCourse($event)">
+        </app-course-item>
+    </li>
+    <button class="coursesLoadMore" (click)="onClickLoadMore()">LOAD MORE</button>
```

Comment:
In html it is better not to write in uppercase, good practice to move it to css text-transform: uppercase
And try to reuse the component of button here :)

Reply:
Done, reused the button component with different initial styling in it